scripts/build.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
```

Comment:
The merit of creating `scripts/build.sh` was that user can use the script file to build easily on local environment.
But if you do not like it, I will remove the script file and write the logic in `.travis.yml`.